common/src/main/java/com/revenuecat/purchases/common/diagnostics/DiagnosticsCounterName.kt

```diff
@@ -0,0 +1,5 @@
+package com.revenuecat.purchases.common.diagnostics
+
+enum class DiagnosticsCounterName {
+    HTTP_REQUEST_PERFORMED,
```

Comment:
In a followup PR, we are planning to track some data as counters, as well as through events